training/distributed_training/pytorch/model_parallel_v2/shared-scripts/train_lib.py

```diff
@@ -110,6 +119,30 @@ def reduce_loss(loss):
     loss_scalar = loss_detached.item() / dist.get_world_size()
     return loss_scalar
 
+def get_batch_for_cp_rank(batch):
+    # Based on https://tiny.amazon.com/1bcmbuhje/githNVIDNeMoblob58d6nemocoll
```

Comment:
Replace this amazon based tiny url with generic url

Reply:
Updated!